include/ringmesh/io/geomodel_builder_gocad.h

```diff
@@ -545,7 +545,7 @@ namespace RINGMesh
     /*!
      * @brief Builds a meshed GeoModel from a Gocad TSolid (file.so)
      */
-    class RINGMESH_API GeoModelBuilderTSolid final : public GeoModelBuilderGocad
+    class io_api GeoModelBuilderTSolid final : public GeoModelBuilderGocad
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "public virtual" or "protected" destructor to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1235)